Description: Update make.conf
 Weird Make.conf needs set up.
 .
 fbv (1.0b-1) unstable; urgency=low
 .
   * Initial release
Author: nick black <nick.black@sprezzatech.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: no
Reviewed-By: Nick Black <nick.black@sprezzatech.com>
Last-Update: 2012-07-07

--- fbv-1.0b.orig/Make.conf
+++ fbv-1.0b/Make.conf
@@ -1,6 +1,2 @@
-prefix	= /usr
-bindir	= /usr/bin
-mandir	= /share/man
-infodir	= /share/info
-
-LIBS	= -lpng -ljpeg -lungif 
+-e error:
+	@echo Please run ./configure first...
